sql/planbuilder/from.go

```diff
@@ -635,7 +635,7 @@ func (b *Builder) buildTablescan(inScope *scope, db, name string, asof *ast.AsOf
 			outScope.newColumn(scopeColumn{
 				db:       strings.ToLower(db),
 				table:    strings.ToLower(name),
-				col:      strings.ToLower(c.Name),
+				col:      c.Name,
```

Comment:
this needs to be `originalCol`

Reply:
Thanks for pointing out the `originalCol` field; I was a little skeptical about changing `col`. Fixed in d0d5527a159ad8c7c008825cede2775e80d7c22f.

Reply:
Sorry I was a bit hasty sending this review! The `originalCol` field is used for display purposes by the target projection. Differentiating the case sensitive and case insensitive might be overly conservative, but so far it has been useful fixing indexing bugs as a result of poorly matching index keys to filter expressions in a case-insensitive way.